src/controllers/itemDetails/index.js

```diff
@@ -1157,10 +1157,7 @@ function renderMoreFromArtist(view, item, apiClient) {
 
     if (section) {
         if (item.Type === 'MusicArtist') {
```

Comment:
It looks like this `if` block can just be removed now.

Reply:
I presume it would need folding into the next if, so something like?
```js
if (item.Type !== 'MusicArtist' && (item.Type !== 'MusicAlbum' || !item.AlbumArtists || !item.AlbumArtists.length)) {
```
holy length batman!

Reply:
Yeah that looks right. We don't enforce a line length limit (yet) so you can find some that are rather long. 😅